curriculum/js0/1.js

```diff
@@ -6,9 +6,11 @@
  */
 
 const solution = (a, b) => {
-  return 0;
+  return a+b;
```

Comment:
This works. One recommendation (for the future), please add spaces around the `+` operator.